submissions/dyworld-history/dyworld-history.md

```diff
@@ -0,0 +1,49 @@
+# DyWorld-Dynamics and it's history <author>Dysoch</author>
+
+Hello everybody, my name is Dysoch. In this article I will write a bit about my history with modding Factorio, why I am doing it and what is coming. So, lets get to the history
+
+## History ##
+When I bought Factorio, I was already playing it for a year, but officialy bought it when Factorio 0.7 was in development. I simply loved the game, but felt it could be better. So I started modding. I had no idea what I was doing, but luckily a nice guy by the name of FreeER came to aid me, and learned me to code properly. I had several breaks over the years where I was burned out on modding or had too much to do in my life, but I always have an itch to come back.
+
+When I started to mod, there was only 1 major overhaul mod, called F-Mod. I wanted to have the same as that mod, and started. It was clunky in the beginning, but eventually DyTech touched every aspect of the game. I rewrote it several times under somewhat the same name, but the content stayed mostly the same. By rewriting, I learned to code it better, easier to read and edit things. I always had several ideas that were a little out of reach for me, but I always strived to make planets into Factorio since I started modding. Luckily Earendel did that now, and I can work on other projects
+
+The DyTech was very populair back in the day. I even played together a few times with Xterminator and Oninaig while they were doing a lets play on Youtube. After a few years (Factorio 0.15) I started to work on the next step of DyTech, DyWorld. A major rewrite after 2 years of no modding, to fit DyTech better into the ever evolving Factorio. I made great strides, and worked on it till just before Factorio 1.0.0. Several content creators played it on Youtube:
```

Comment:
Could you add a link the lets plays?

And could you expand a little on what was different of DyWorld?